docs/data_node/extract_node/oracle-cdc.md

```diff
@@ -176,6 +176,7 @@ Overall, the steps for configuring CDB database is quite similar to non-CDB data
      GRANT SELECT ANY TRANSACTION TO flinkuser CONTAINER=ALL;
      GRANT LOGMINING TO flinkuser CONTAINER=ALL;
      GRANT CREATE TABLE TO flinkuser CONTAINER=ALL;
+     -- need not to execute if set scan.incremental.snapshot.enabled=true(default)
```

Comment:
Don’t need to execute the following statement, If you set 'scan.incremental.snapshot.enabled=true' (default).

Reply:
Thanks you, it has been updated.